projects/maixpy_m5stickv/compile/overrides/components/micropython/port/src/omv/img/font.c

```diff
@@ -44,7 +44,7 @@ static uint8_t unicode[] = {
 0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
 0x00,0x00,0x10,0x10,0x10,0x10,0x10,0x10,0x10,0x00,0x10,0x10,0x00,0x00,
 0x00,0x24,0x24,0x24,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
-0x00,0x00,0x24,0x24,0x24,0x7E,0x24,0x24,0x7E,0x24,0x24,0x24,0x00,0x00,
+0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,0xFF,
```

Comment:
Instead of replacing the glyph for an existing ASCII character that we might want or need to use, I propose the same solution here as I did [here](https://github.com/selfcustody/krux/issues/111) with the bitcoin symbol, finding an unused character from the [Latin-1 Supplement block](https://www.compart.com/en/unicode/block/U+0080). In this case, U+0080 looks like a good fit.